infra-templates/longhorn/0/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '2'
+services:
+  longhorn-driver:
+    privileged: true
+    pid: host
+    image: imikushin/storage-longhorn:dev45
+    labels:
+      io.rancher.scheduler.global: 'true'
+      io.rancher.container.create_agent: 'true'
+      io.rancher.container.agent.role: 'environment'
+    volumes:
+    - /run:/run
+    - /var/run:/var/run
+    - /dev:/host/dev
+    - /var/lib/rancher/volumes:/var/lib/rancher/volumes:shared
+    - /var/lib/rancher/longhorn:/var/lib/rancher/longhorn
+    environment:
+      RANCHER_DEBUG: ${RANCHER_DEBUG}
+    logging:
+      driver: json-file
+      options:
+        max-size: 25m
+        max-file: '2'
+
+  longhorn-orc:
+    image: imikushin/longhorn-orc:8d72167
+    labels:
+      io.rancher.sidekicks: longhorn-binary
+      io.rancher.scheduler.global: 'true'
+      io.rancher.container.create_agent: 'true'
+      io.rancher.container.agent.role: 'environment'
+    volumes:
+    - /var/run:/var/run
+    volumes_from:
+    - longhorn-binary   # /usr/local/bin
+    environment:
+      RANCHER_DEBUG: ${RANCHER_DEBUG}
+      LONGHORN_IMAGE: imikushin/longhorn:bca578f
+
+  longhorn-binary:
+    image: imikushin/longhorn:bca578f
```

Comment:
@imikushin `rancher/longhorn:bca578f` is available now as latest Longhorn image.